viewer/README.md

```diff
@@ -43,3 +43,65 @@ In case if you see an error like that:
 > Refused to load the script 'https://apps-cdn.cogniteapp.com/@cognite/reveal-parser-worker/1.1.0/reveal.parser.worker.js' because it violates the following Content Security Policy directive: "script-src 'self' https://localhost:* blob:"
 
 See the next steps in our [documentation](https://cognitedata.github.io/reveal-docs/docs/installation#installation-for-projects-with-content-security-policy).
+
+## Local Packages
+The Reveal viewer is structured using local packages.
+This allows you to test features in isolation and constrain the dependencies with logical barriers between features.
+Packages are located in the `/packages/` subfolder.
+There is no enforced structure of the packages and their layout and content will vary depending on what the package exposes to the rest of the system.
```

Comment:
I think we however should at least suggest that it follows a standard, e.g. 
- `src/` - source files and tests
- `app/` - test application with `index.ts` as entry
- `index.ts` as main entry.
I just think it would benefit us if we had one fixed structure that we follow (the above is just a half-ass example that doesn't probably cover everything that we should "standarize")